src/dso/cli/__init__.py

```diff
@@ -266,13 +267,56 @@ def command_wrapper(args):
     return command_wrapper
 
 
+@click.command(name="mv")
+@click.argument(
+    "source",
+    type=click.Path(exists=True, file_okay=True, dir_okay=True, path_type=Path),
+)
+@click.argument(
+    "target",
+    type=click.Path(file_okay=True, dir_okay=True, path_type=Path),
+    required=False,
+)
+@click.option(
+    "--increment-prefix",
```

Comment:
```suggestion
    "--increment-prefix",
    "-i",
```

increment-prefix is a pain to type, let's add an additional shorthand option